### PR TITLE
Document syntax changes from older versions to deal with outdated tutorials

### LINKED ISSUE #176: Changelog for non backward-compatible changes
Should be somewhere in the manuals, short. Could be used to update eg. NAMD tutorials. Reference in case something goes wrong.

-> lower/upperBoundary -> lower/upperWall
-> refPosGroup -> fittingGroup

### PR DESCRIPTION
**Pull request by giacomofiorin**
Fixes #176

### CODE REVIEW

**Comment by giacomofiorin** [comment]
@jhenin I'm going to request also @jribeiroUIUC to take a look before merging, to minimize legwork in sending this change to NAMD Gerrit and back.

File: doc/colvars-refman-main.tex [hunk 1]
```diff
@@ -4660,3 +4663,37 @@ \subsection{LAMMPS keywords}
 
 }
 
+
+\cvsec{Syntax changes from older versions}{sec:colvars_config_changes}
+
+The following is a list of syntax changes in Colvars since its first release.
+Many of the older keywords are still recognized by the current code: the primary purpose of this list is to let you aware of improvements that affect old configuration files.
```
Comment (jhenin) [inline doc/colvars-refman-main.tex]: let you aware -> make you aware

File: doc/colvars-refman-main.tex [hunk 2]
```diff
@@ -4660,3 +4663,37 @@ \subsection{LAMMPS keywords}
...
+
+The syntax of the latest version of Colvars can always be accessed at:\\
+\cvnamdonly{\url{https://colvars.github.io/colvars-refman-namd/colvars-refman-namd.html}}
+\cvlammpsonly{\url{https://colvars.github.io/colvars-refman-lammps/colvars-refman-lammps.html}}
+\cvvmdonly{\url{https://colvars.github.io/colvars-refman-vmd/colvars-refman-vmd.html}}
+
+If you are using any of the NAMD and VMD tutorials:\\
+\url{https://www.ks.uiuc.edu/Training/Tutorials/}\\
+please be aware that \emph{several of these tutorials have not been actively maintained}: while they are updated, the following list will help you reconcile any inconsistencies.
```
Comment (jhenin) [inline doc/colvars-refman-main.tex]: while they are updated -> pending their update

**Comment by jribeiroUIUC** [comment]
For me it is good to go as well. Incorporate these messages (or similar) in the tutorials will also help to convey the updates in the colvars syntax. Thank you.

File: doc/colvars-refman-main.tex (hunk 2 above)
Comment (giacomofiorin) [reply doc/colvars-refman-main.tex]: I already changed this part in the next commit, and made it more generic (we cannot be sure whether the tutorial under consideration will be updated).

**Pull request merged**